pages/common/colorpicker.md

```diff
@@ -0,0 +1,24 @@
+# colorpicker
+
+> A minimalist X11 colorpicker.
+> More information: <https://github.com/ym1234/colorpicker>.
+
+- Launch colorpicker and print the HEX and RGB value of each clicked pixel to stdout:
```

Comment:
We should maybe somehow include that any mouse action excipt left click exits the program. This thing is almost harder to exit than vim.

Reply:
`:x`
`:ZZ`
`q`
`q!`

I joke of course. Vim to total beginners is odd, me included.

Maybe we could have in the description, `Press ___ to exit`

Reply:
I think it would be best to include `> Any mouse gesture except left click will exit the program` in the command description.

Reply:
Resolved as the command's description has been updated.